include/DoocsProcessScalar.h

```diff
@@ -81,22 +81,11 @@ namespace ChimeraTK {
       //
       // We should also checked if data should be stored (flag storeInHistory). Invalid data should NOT be stored except first invalid data point.
```

Comment:
The whole comment can go away. The concept of not writing to history does not exist any more, and also storeInHistory is not there any more (commented code should be removed).